bot2.js

```diff
@@ -11,158 +11,186 @@ const config = require("./config.json");
+            break;
+        }
+
+        case 'p1jedi':
+        case 'yoda':
+        case 'gmyoda': {
+            message.channel.send('https://drive.google.com/file/d/1l9_kGeyo8XzdEhzwyeoiHmK27TZ7z__-/view?usp=drivesdk');
+            break;
+        }
+
+        case 'p2jedi':
+        case 'bastilla': {
+            message.channel.send('https://drive.google.com/open?id=1Z06JomJBransHh8MDl7ikQKsn_J383st');
+            break;
+        }
+
+        case 'leia':
+        case 'machinegunleia': {
+            message.channel.send('https://drive.google.com/open?id=1zRxBz7r5qOS-G9WCsz3PHHSz6VRNmLzt');
+            break;
+        }
+
+        case 'nute':
+        case 'churchofnute': {
+            message.channel.send('https://drive.google.com/open?id=1p3w7EFQHNllxcvGP5uE37xaKgJLe-V02');
+            break;
+        }
+
+        case 'phoenix': {
+            message.channel.send('https://drive.google.com/open?id=1FhGBkqX4LOueRMKxVs5dIXvLAEL7PhB9');
+            break;
+        }
+
+        case 'chexmix': {
+            message.channel.send('https://drive.google.com/open?id=1Pg1goUW-bRebWd8nzJ9es8nF0Nk4S9KC');
+            break;
+        }
+
+        case 'yolorolo': {
+            message.channel.send('https://drive.google.com/open?id=1fWZPMJZ1YDDgbyauIC4ByocEaTZ8ng9V');
+            break;
+        }
+
+        case 'nightsisters':
+        case 'nightsister': {
+            message.channel.send('https://drive.google.com/open?id=1p5LlfkMVn8G-c8I6NusdWoMKGNozTXyl');
+            break;
+        }
+
+        case 'nihilusmatrix': {
+            message.channel.send('https://drive.google.com/open?id=1k9pxWm2MwwuiJhxCDRdK-TGMyMg92mgd');
+            break;
+        }
+
+        case 'p1': {
+            let num = Number(args[0].replace(/\,/g, '')) / 46888776 * 100;
+            message.channel.send(Math.round(num * 10) / 10 + "%");
+            break;
+        }
+
+        case 'p2': {
+            let num = Number(args[0].replace(/\,/g, '')) / 52105585 * 100;
+            message.channel.send(Math.round(num * 10) / 10 + "%");
+            break;
+        }
+
+        case 'p3': {
+            let num = Number(args[0].replace(/\,/g, '')) / 38371455 * 100;
+            message.channel.send(Math.round(num * 10) / 10 + "%");
+            break;
+        }
+
+        case 'p4': {
+            var num = Number(args[0].replace(/\,/g, '')) / 33499537 * 100;
+            message.channel.send(Math.round(num * 10) / 10 + "%");
+            break;
+        }
+
+        case 'invite': {
+            message.channel.send("Use the following link (need server admin permissions): https://discordapp.com/oauth2/authorize?client_id=476374210122612747&scope=bot . For support, contact TheNo0b#9329 on Discord");
+            break;
+        }
+
+        case 'readiness': {
+            message.channel.send("coming very soon, stay tuned!")
+            break;
+        }
+
+        default: {
+            message.channel.send("invalid command");
+        }
+    }
+
```

Comment:
I'm a big advocate of switch statements. This looks a lot cleaner and should be a lot faster than going through all of those if statements. There was no return or breaks, so it was hitting every single if statement. Since we only have a single command that can be ran at a time, it's best to just run a switch.